common/lib/xmodule/xmodule/tests/test_sequence.py

```diff
@@ -92,7 +94,12 @@ def _set_up_module_system(self, block):
         module_system.descriptor_runtime = block._runtime  # pylint: disable=protected-access
         block.xmodule_runtime = module_system
 
-    def _get_rendered_student_view(self, sequence, requested_child=None, extra_context=None, self_paced=False):
+    def _get_rendered_student_anonymous_view(self,
```

Comment:
Since now you are passing the view as a parameter, you can call it `_get_rendered_view()`.